AdminWebsite/AdminWebsite/ClientApp/src/app/testing/stubs/search-email-stub.ts

```diff
@@ -1,10 +1,10 @@
 import { Component, Output, ViewChild, ElementRef, EventEmitter } from '@angular/core';
-import { ParticipantModel } from '../../common/model/participant.model';
+import { VHParticipant } from 'src/app/common/model/vh-participant';
```

Comment:
I've found ngMocks useful in testing that has dependencies on other components such as this. If it's an easy removal, then it would be nice to see the back of this

Reply:
good idea, could I suggest a separate ticket for that though. There are quite a few instances of stubs which would be to good to replace en masse rather than making this PR bigger